src/cmd/internal.go

```diff
@@ -51,7 +55,11 @@ var httpProxyCmd = &cobra.Command{
 	Short: lang.CmdInternalProxyShort,
 	Long:  lang.CmdInternalProxyLong,
 	RunE: func(cmd *cobra.Command, _ []string) error {
-		return agent.StartHTTPProxy(cmd.Context())
+		cluster, err := cluster.NewCluster()
```

Comment:
This path didn't previously have a `cluster.NewCluster`, so beforehand if there was none it would error?

Reply:
The cluster was just created further up the  chain. This makes things more clear and easier to test.